src/SFA.DAS.EAS.Portal/Extensions/StringExtensions.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Net;
+
+namespace SFA.DAS.EAS.Portal.Extensions
+{
+    public static class StringExtensions
+    {
+        public static string HtmlDecode(this string input)
```

Comment:
Doesn't the SFA.DAS.NServiceBus.EndpointConfigurationExtensions handle the encoding of the license?  Maybe it should.